ibm/mas_devops/roles/cp4d_install_services/templates/watsonstudiocfg.yml.j2

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ mas_instance_id }}-usersupplied-watsonstudio-creds-system"
+  namespace: "mas-{{mas_instance_id}}-core"
+stringData:
+  password: "{{ cpd_admin_password | b64decode }}"
+  username: "{{ cpd_admin_username }}"
+---
+apiVersion: config.mas.ibm.com/v1
+kind: WatsonStudioCfg
+metadata:
+  name: "{{mas_instance_id}}-watsonstudio-system"
+  namespace: "mas-{{mas_instance_id}}-core"
+  labels:
+    mas.ibm.com/configScope: system
+    mas.ibm.com/instanceId: "{{mas_instance_id}}"
+spec:
+  config:
+    credentials:
+      secretName: "{{ mas_instance_id }}-usersupplied-watsonstudio-creds-system"
+    endpoint: "{{ cpd_url }}"
+  displayName: "MAS - Watson Studio configuration"
+  type: external
+  
```

Comment:
Github reports no newline at end of file ... current linting lets this pass for j2 files, but future won't, so please add missing newline

Reply:
I thought row 26 was enough as extra new line? in any case i added one more... is it ok now?